tests/test_input.py

```diff
@@ -0,0 +1,385 @@
+                        "button": 0,
+                    }, {
+                        "type": "pointerMove",
+                        "x": 1,
+                        "y": 1,
+                    }, {
+                        "type": "pointerUp",
+                        "button": 0,
+                    }]
+                }]
+            }
+        })
+
+    result = await execute_command(
+        websocket, {
+            "method": "script.evaluate",
+            "params": {
+                "expression": "allEvents",
+                "awaitPromise": False,
+                "resultOwnership": "root",
+                "target": {
+                    "context": context_id
+                }
+            }
+        })
+
+    assert {
+        'type': 'success',
+        'result': {
+            'type': 'array',
+            'value': [{
+                'type': 'object',
+                'value': [['event', {
+                    'type': 'string',
+                    'value': 'mousedown'
+                }], ['button', {
+                    'type': 'number',
+                    'value': 0
+                }], ['buttons', {
+                    'type': 'number',
+                    'value': 1
+                }], ['clientX', {
+                    'type': 'number',
+                    'value': 0
+                }], ['clientY', {
+                    'type': 'number',
+                    'value': 0
+                }]]
+            }, {
+                'type': 'object',
+                'value': [['event', {
+                    'type': 'string',
+                    'value': 'mousemove'
+                }], ['button', {
+                    'type': 'number',
+                    'value': 0
+                }], ['buttons', {
+                    'type': 'number',
+                    'value': 1
+                }], ['clientX', {
+                    'type': 'number',
+                    'value': 1
+                }], ['clientY', {
+                    'type': 'number',
+                    'value': 1
+                }]]
+            }, {
+                'type': 'object',
+                'value': [['event', {
+                    'type': 'string',
+                    'value': 'mouseup'
+                }], ['button', {
+                    'type': 'number',
+                    'value': 0
+                }], ['buttons', {
+                    'type': 'number',
+                    'value': 0
+                }], ['clientX', {
+                    'type': 'number',
+                    'value': 1
+                }], ['clientY', {
+                    'type': 'number',
+                    'value': 1
+                }]]
+            }],
+            'handle': ANY_STR
+        },
+        'realm': ANY_STR
+    } == result
+
+
+# TODO(jrandolf): Investigate mouse wheel flakiness. The deltaY sometimes
+# doubles and other times not. It is also not independent of the mouse.
+# @pytest.mark.asyncio
```

Comment:
Please use @pytest.mark.skip(reason="TODO: Implement") instead of submitting commented out code